src/parsers/xyz.js

```diff
@@ -140,10 +144,37 @@ function fromMaterial(materialOrConfig, fractional = false) {
     return fromBasis(basis, "%11.6f");
 }
 
+/**
+ * Function splits the xyzFile string at new lines and then returns the first element of the array.
+ * The first line of the XYZ file should contain the number of atoms in the structure.
+ * @param {String} xyzFile
+ * @returns {Number}
+ */
+export function xyzFileAtomsCount(xyzFile) {
+    return Integer.parseInt(xyzFile.split(/\r?\n/)[0]);
+}
+
+/**
+ * Function converts an XYZ formatted structure as a POSCAR formatted structure
+ * @param {String} xyzContent
+ * @returns {String}
+ */
+export function xyzToPoscar(xyzContent) {
+    const xyzConfig = defaultMaterialConfig;
+    const xyzArray = xyzContent.split(/\r?\n/);
+    const xyzArrayBasisOnly = xyzArray.slice(2, -1);
+    const xyzBasis = xyzArrayBasisOnly.join("\n");
+    xyzConfig.basis = toBasisConfig(xyzBasis);
+    xyzConfig.basis.units = "cartesian";
+    return poscar.toPoscar(xyzConfig);
```

Comment:
The placement of this function seems to be bad at the moment, hence the circular dependency. We should move this functionality to resolve the problem - maybe to top-level of parsers?